lib/App/index.js

```diff
@@ -132,10 +132,15 @@ class App {
       }).catch(err => {
         Log(colors.red('✖', err.message || err.toString()));
       })
-    activeHomey.devkit.on('disconnect', () => {
-      Log(colors.red(`✖ Connection has been lost, exiting...`));
-      process.exit();
-    })
+
+      activeHomey.devkit.on('disconnect', () => {
+        Log(colors.red(`✖ Connection has been lost, attempting to reconnect...`));
+
+        // reconnect event isn't forwarded from athom api
+        activeHomey.devkit.once('connect', () => {
+          Log(colors.green(`✓ Reconnected, some app logs may have been lost`));
```

Comment:
```suggestion
          Log(colors.green(`✓ Connection restored, some output might be missing`));
```

Reply:
logging sounds like the action to me

> ## Logging
> - The process of recording actions that take place on a computer, network, or system.
> - The work of cutting down trees for timber.

I don't think it matters much either way so I changed it to "logs" (still makes me think of multiple log files and or tree trunks :P)

Reply:
Haha I get your point, like I said, might be a personal thing or just plain wrong:p If you want to go with your initial idea that is fine as well!